### PR TITLE
Feature#18 pdf csv generate

### PR DESCRIPTION
**Pull request by jnegrete15**
#PDF and CSV generators

User can download their task in both pdf and csv file

### PDF File:
Screenshot from 2021-05-18 16-16-02

### CSV  File:
Screenshot from 2021-05-18 16-16-38

### CODE REVIEW

**Review by OchoaHub — APPROVED** [review]
Espero no tenga conflictos porque ya no apruebo dos veces

File: Gemfile [hunk 1]
```diff
@@ -59,4 +59,6 @@ gem 'clearance', '~> 2.4'
 gem 'rubocop', require: false
 gem 'rubycritic', require: false
 gem 'sandi_meter', '~> 1.2'
+gem 'wkhtmltopdf-binary'
+gem 'wicked_pdf', git: 'https://github.com/mileszs/wicked_pdf.git', branch: 'master'
```
Comment (OchoaHub) [inline Gemfile:63]: I like your gems choice!

**Review by LPMendoza — APPROVED** [review]
Good job, that gem is very useful

File: app/views/main/index.html.erb [hunk 2]
```diff
@@ -38,6 +38,12 @@
         <% end %>
       </tbody>
     </table>
+
+    <div class="btn-convert-container">
+      <%= link_to "CSV File", '/report.csv', class: 'btn btn-pdf' %>
+      <%= link_to "PDF File", '/report.pdf', :target => "_blank" , class: 'btn btn-pdf' %>
```
Comment (OchoaHub) [inline app/views/main/index.html.erb:44]: Nice links!

**Review by OchoaHub — APPROVED** [review]
Without conflicts! Good

**Pull request merged**